spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubTemplateIntegrationTests.java

```diff
@@ -168,8 +173,8 @@ public void testPullAndAck() {
 						message.nack().get(); // sync call
 					}
 				}
-				catch (InterruptedException | ExecutionException e) {
-					e.printStackTrace();
+				catch (InterruptedException | ExecutionException ex) {
+					LOGGER.error(ex);
```

Comment:
But you still don't `interrupt()`...

Reply:
Although this is a test code, so never mind.
Let's merge!

Reply:
sorry, forgot, it's set now